stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/gallery/overview/MediaAttachmentGridView.kt

```diff
@@ -80,9 +85,12 @@ public class MediaAttachmentGridView : FrameLayout {
     }
 
     private fun init(context: Context, attrs: AttributeSet?) {
```

Comment:
I think the code above is incorrect. The Context passed to the `init` method is not wrapped. Could you check the scenario below? 

```
<style name="StreamUiTheme" parent="Theme.MaterialComponents.DayNight.NoActionBar">
     ...
     <item name="streamUiMediaAttachmentGridViewStyle">@style/StreamUi.MediaAttachmentGridView</item>
</style>     
     
     
<style name="StreamUi.MyCustomMediaAttachmentGridView">
        <!-- This guy is different -->
        <item name="streamUiMediaAttachmentGridViewPlayVideoButtonIcon">@drawable/stream_ui_ic_umnute</item>

        <item name="streamUiMediaAttachmentGridViewPlayVideoIconElevation">3dp</item>
        <item name="streamUiMediaAttachmentGridViewPlayVideoIconPaddingTop">2dp</item>
        <item name="streamUiMediaAttachmentGridViewPlayVideoIconPaddingBottom">2dp</item>
        <item name="streamUiMediaAttachmentGridViewPlayVideoIconPaddingStart">3dp</item>
        <item name="streamUiMediaAttachmentGridViewPlayVideoIconPaddingEnd">2dp</item>
        <item name="streamUiMediaAttachmentGridViewPlayVideoIconCornerRadius">2dp</item>
        <item name="streamUiMediaAttachmentGridViewPlayVideoIconBackgroundColor">@color/stream_ui_literal_white</item>
        <item name="streamUiMediaAttachmentGridViewImagePlaceholder">@drawable/stream_ui_picture_placeholder</item>
    </style>    
```